runtime/doc/syntax.txt

```diff
@@ -5991,6 +5991,12 @@ TabLine		Tab pages line, not active tab page label.
 TabLineFill	Tab pages line, where there are no labels.
 							*hl-TabLineSel*
 TabLineSel	Tab pages line, active tab page label.
+							*hl-TabSideBar*
+TabSideBar	tabsidebar, not active tab page label
+							*hl-TabSideBarFill*
+TabSideBarFill	tabsidebar, where there are no labels
+							*hl-TabSideBarSel*
+TabSideBarSel	tabsidebar, active tab page label
```

Comment:
It would be better to add "." based on the existing text.

```suggestion
TabSideBar	tabsidebar, not active tab page label.
							*hl-TabSideBarFill*
TabSideBarFill	tabsidebar, where there are no labels.
							*hl-TabSideBarSel*
TabSideBarSel	tabsidebar, active tab page label.
```